addon/components/feature-result-item.js

```diff
@@ -12,15 +12,28 @@ import layout from '../templates/components/feature-result-item';
   @extends <a href="http://emberjs.com/api/classes/Ember.Component.html">Ember.Component</a>
  */
 export default Ember.Component.extend({
+
+  classNames: ['item', 'feature-result-item'],
+
+  classNameBindings: ['isActive:active'],
+
   /**
-    Flag: indicates where display detailed feature info
+    Flag: indicates whether to display detailed feature info.
     @property _infoExpanded
     @type boolean
     @default false
    */
   _infoExpanded: false,
 
   /**
+    Flag: indicates whether to display links list (if present).
+    @property _linksExpanded
```

Comment:
Comment's description and its "body" must be saparated with an empty line.
Same remark is true for the other class members.